packages/babel-preset-env/test/fixtures/corejs2/usage-symbol-iterator-in/output.mjs

```diff
@@ -1,4 +1,6 @@
-import "core-js/modules/es7.symbol.async-iterator.js";
-import "core-js/modules/es6.symbol.js";
+import "core-js/modules/es6.string.iterator.js";
+import "core-js/modules/es6.object.to-string.js";
+import "core-js/modules/es6.array.iterator.js";
```

Comment:
The original output seems better. I am not sure why array iterator should be pulled here.

Reply:
It's because `Symbol.iterator in arr` should return `true` if `arr` is an array, and this can only happen if we load the `core-js/modules/es6.array.iterator.js` polyfill.

Our core-js 3 logic was already smart in this case (I reused it for core-js 2), and it already loads that polyfill on `main`: https://babeljs.io/repl#?browsers=defaults&build=&builtIns=usage&spec=false&loose=false&code_lz=MoTwtgRg9gNgdASwC4FMBOBDJU0AIEB2uGaaA3EA&debug=false&forceAllTransforms=false&shippedProposals=true&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=&version=7.12.12&externalPlugins=